### PR TITLE
chore(integration-templates): [nan-1170] remove integration templates

### PR DESCRIPTION
**Pull request by khaliqgant**
## Describe your changes
Remove integration templates to use the new location https://github.com/NangoHQ/integration-templates

## Issue ticket number and link
NAN-1170

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,8 +42,6 @@
         "dwa": "npm run dev:watch:apps",
         "test": "concurrently --kill-others \"npm run test:unit\" \"npm run test:integration\" \"npm run test:cli\" \"npm run test:integration-templates\"",
         "test:unit": "vitest",
-        "test:integration": "vitest --config ./vite.integration.config.ts",
```
Comment (bodinsamuel) [inline package.json:46]: woops

**Review by bodinsamuel — APPROVED** [review]
Screenshot 2024-09-26 at 16 09 10

:feels_good:

**Review by TBonnin — APPROVED** [review]
looks good other than the removal of `test:integration` ;)

File: package.json (hunk 1 above)
Comment (khaliqgant) [reply package.json:46]: 😱

updated

**Pull request merged**